tests/testutils/_primer/test_primer.py

```diff
@@ -20,39 +22,70 @@
 
 PRIMER_CURRENT_INTERPRETER = (3, 10)
 
+DEFAULT_ARGS = ["python tests/primer/__main__.py", "compare", "--commit=v2.14.2"]
```

Comment:
```suggestion
DEFAULT_ARGS = ["python tests/primer/__main__.py", "compare", "--commit=v2.14.2"]
```

Should this bump be included in `tbump` or can we use `gitpython` to detect the latest tag?

Reply:
It does not matter as the result of the command is in the json we provide in the fixtures with ``[f"--base-file={main}", f"--new-file={pr}"]``. In fact I think it's only a string we're using in order to have the information in the comment message at the end.